golem_messages/message.py

```diff
@@ -1358,6 +1358,27 @@ def deserialize_slot(self, key, value):
         return deserialize_task_to_compute(key, value)
 
 
+class FileTransferToken(Message):
+    TYPE = CONCENT_MSG_BASE + 5
+
+    __slots__ = [
+        'token_expiration_deadline',
+        'storage_cluster_address',
+        'authorized_client_public_key',
+        'operation',
+        'files_data',
+    ] + Message.__slots__
+
+
+class TokenFilesData(datastructures.FrozenDict):

```

Comment:
Since this class is tightly related to `FileTransferToken` could it be it's subclass? Similar to enums in other messages (e.g. `ServiceRefused.Reason`)?